packages/node_modules/@ciscospark/react-component-adaptive-card/src/index.js

```diff
@@ -1,14 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import ReactDOM from 'react-dom';
 import classnames from 'classnames';
-import {getAdaptiveCard} from '@ciscospark/react-component-utils';
+import {getAdaptiveCard, CARD_CONTAINS_IMAGE, replaceIndexWithBlobURL, API_ACTIVITY_VERB} from '@ciscospark/react-component-utils';
 
 class AdaptiveCard extends React.Component {
   constructor(props) {
     super(props);
     this.nodeElement = React.createRef();
     this.state = {
-      childNodes: []
+      childNodes: [],
+      hasReplacedImagesInJSON: false,
```

Comment:
I don't think this should be a component state. There is only one place that this is being set (line 54) and never updated again

Reply:
State variable setting happens every time when props updated. This state variable is using in the condition of render to replace the child node(at line 83).